supplier_server_mock/setup.py

```diff
@@ -8,7 +8,7 @@
 
 setuptools.setup(
     name='Supplier API mock server',
-    version='1.4.3',
+    version='1.4.4',
```

Comment:
**Non blocking**: In tester you've added varsioning from `__version__` file. Can we do something similar here?

Reply:
Yes, but it's not needed here. I did this to be able to import the version in the web tool.